apmpackage/apm/data_stream/service_metrics/manifest.yml

```diff
@@ -0,0 +1,31 @@
+title: APM service metrics
+type: metrics
+dataset: apm.service
+ilm_policy: metrics-apm.service_metrics-default_policy
+elasticsearch:
+  index_template:
+    mappings:
+      # Internal metrics should have all fields strictly mapped;
+      # we are in full control of the field names.
+      dynamic: false
```

Comment:
Some copy pasta here, but also you've changed it from strict to dynamic. Any particular reason why we should allow unmapped fields?